packages/eslint-plugin-internal/package.json

```diff
@@ -14,7 +14,7 @@
   },
   "dependencies": {
     "@types/prettier": "*",
-    "@typescript-eslint/experimental-utils": "5.4.0",
+    "@typescript-eslint/utils": "5.4.0",
```

Comment:
I don't know what to do to get builds passing. Should we manually publish...?

Reply:
Would you mind merging in latest main so that we can do another run, I think things aren't running at all right now because it predates the master->main rename.

It might be a question for Brad as to why the updated integration tests format might be trying to reach out to npm for this, but would be good to have a fresh CI run to reproduce the issue.

(Aside: In angular-eslint, which I set up a lot more recently than typescript-eslint, I run a local registry using verdaccio and leverage lerna to get very close to a real version and publish cycle (which updates all these package.json refs) and publish the packaged form of the HEAD of each package. I've found it to work pretty well, so we might want to consider moving to that kind of setup)